portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body> 
+      <div class="row">
+        <img src="images/logo.png"
+    </div>   
+    <ul class="main-nav">    
+        <li><a href="index.html" style="color:white"> HOME </a></li>
+        <li class="active"><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+		<li><a href="" style="color:white"> PROJECT SHOWCASE </a></li>
+        <li><a href="index2.html" style="color:white"> ABOUT </a></li>
+        <li><a href="contact.html" style="color:white"> CONTACT </a></li>
+    </ul>
+    <!--<div id="content">-->
```

Comment:
NIT: it's usually best to remove commented-out lines before submitting.

By the way, "NIT" means a nitpick - something that's not a big deal, just a small you could change.  Googlers tend to use that a lot in code reviews

Reply:
fixed